src/Domain/NotificationPoll.php

```diff
@@ -0,0 +1,36 @@
+<?php declare(strict_types = 1);
+
+namespace SandwaveIo\RealtimeRegister\Domain;
+
+final class NotificationPoll implements DomainObjectInterface
+{
+    /** @var string */
+    public $count;
+
+    /** @var array<string>|null */
+    public $notification;
+
+    private function __construct(string $count, ?array $notification)
+    {
+        $this->count = $count;
+        $this->notification = $notification;
+    }
+
+    public static function fromArray(array $json): NotificationPoll
+    {
+        return new NotificationPoll(
+            $json['count'],
+            $json['notification'] ?? ''
```

Comment:
This will not work, the documentation says that the notification value is a Notification object. You could reuse the `Notification` class for this.